examples/client/client.c

```diff
@@ -3074,14 +3077,14 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
 #endif /* HAVE_SECURE_RENEGOTIATION */
 
     if (sendGET) {
+        char msgGet[GETMSGSZ] = "GET /index.html HTTP/1.0\r\n\r\n";
         printf("SSL connect ok, sending GET...\n");
-        msgSz = sizeof("GET /index.html HTTP/1.0\r\n\r\n");
-        XSTRNCPY(msg, "GET /index.html HTTP/1.0\r\n\r\n", msgSz);
-        msg[msgSz] = '\0';
 
-        resumeSz = msgSz;
-        XSTRNCPY(resumeMsg, "GET /index.html HTTP/1.0\r\n\r\n", resumeSz);
-        resumeMsg[resumeSz] = '\0';
+        XMEMSET(msg, 0, MSG32);
+        XMEMSET(resumeMsg, 0, MSG32);
+        msgSz = resumeSz = sizeof(msgGet) - 1;
```

Comment:
If using XMEMCPY instead of XSTRNCPY lets match the changes in server.c and use XSTRLEN here